bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/VersionedClientlibsTransformerFactory.java

```diff
@@ -272,10 +274,26 @@ private String getVersionedPath(final String originalPath, final LibraryType lib
     }
 
     private HtmlLibrary getLibrary(LibraryType libraryType, String libraryPath, ResourceResolver resourceResolver) {
-        String resolvedLibraryPath = resolvePathIfProxied(libraryType, libraryPath, resourceResolver);
+        String resolvedLibraryPath = resolvePath(libraryType, libraryPath, resourceResolver);
         return resolvedLibraryPath == null ? null : htmlLibraryManager.getLibrary(libraryType, resolvedLibraryPath);
     }
 
+    private String resolvePath(LibraryType libraryType, String libraryPath, ResourceResolver resourceResolver) {
+        String clientLibraryPath = resolvePathIfJcrMapping(libraryPath, resourceResolver);
+        if (StringUtils.isEmpty(clientLibraryPath)) {
+            clientLibraryPath = resolvePathIfProxied(libraryType, libraryPath, resourceResolver);
+        }
+        return clientLibraryPath;
+    }
+
+    private String resolvePathIfJcrMapping(String libraryPath, ResourceResolver resourceResolver) {
+        Resource libraryResource = resourceResolver.resolve(libraryPath);
```

Comment:
Please use the two argument version, as otherwise resolve may fail (as things like request headers cannot be taken into account): https://sling.apache.org/apidocs/sling11/org/apache/sling/api/resource/ResourceResolver.html#resolve-javax.servlet.http.HttpServletRequest-java.lang.String-

Reply:
I passed the request through the methods till the new one that I added